docs/Instructional gifs and minting process instructions

```diff
@@ -0,0 +1,54 @@
+## Minting a GitPOAP
```

Comment:
@yalexis-eth - can you rename this file to `instructions.md`? 

the `.md` is pretty important.

Reply:
Hey, sure thing! I'm learning my way in Github to contribute the best I can.